src/components/CreateAgent/Persona/UploadAvatarImage.tsx

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import classNames from "classnames";
+import { toast } from "react-toastify";
+
+import CameraIcon from "src/assets/camera.svg?react";
+import tickSolid from "images/tickSolid.png";
+
+type UploadAvatarImageProps = {
+  selected: boolean;
+  customImageNameRef: React.MutableRefObject<string>;
+  onAvatarSelected?: (image: string) => void;
+};
```

Comment:
this should not be optional right?

Reply:
True, changing it, had made it optional during dev for some tests